tools/src/main/java/org/apache/kafka/trogdor/common/WorkerUtils.java

```diff
@@ -134,35 +135,12 @@ public static void createTopics(
         }
     }
 
-    /**
-     * Returns a list of all existing topic partitions  that match the following criteria: topic
-     * name matches give regular expression 'topicRegex', topic is not internal, partitions are
-     * in range [startPartition, endPartition]
-     *
-     * @param log                The logger to use.
-     * @param bootstrapServers   The bootstrap server list.
-     * @param topicRegex         Topic name regular expression
-     * @param startPartition     Starting partition of partition range
-     * @param endPartition       Ending partition of partition range
-     * @return List of topic partitions
-     * @throws Throwable If getting list of topics or their descriptions fails.
-     */
-    public static Collection<TopicPartition> getMatchingTopicPartitions(
```

Comment:
This is a public method in a public class ... are you sure we can delete it?

Reply:
This method is not a public API.  It's fine to delete it.  Thanks for asking.